app/page/template/list/conversations.htm

```diff
@@ -1,157 +1,159 @@
 <div id="conversations" class="conversations left-list-is-visible" data-bind="with: $root.conversations">
 
-  <!-- ko foreach: conversations_calls -->
-    <div class="calls" data-uie-name="status-call">
-      <div class="calls-avatar">
-        <!-- ko ifnot: call().is_declined() -->
-          <!-- ko if: is_one2one() -->
-            <!-- ko if: call().participants()[0]-->
-              <user-avatar class="user-avatar-xs" params="user: call().participants()[0].user"></user-avatar>
-            <!-- /ko -->
-          <!-- /ko -->
-          <!-- ko if: is_group() && call().state() != z.calling.enum.CallState.ONGOING -->
-            <user-avatar class="user-avatar-xs" params="user: call().creator()"></user-avatar>
-          <!-- /ko -->
-        <!-- /ko -->
-        <!-- ko if: is_group() && (call().state() === z.calling.enum.CallState.ONGOING && !call().self_user_joined()) || call().is_declined() -->
-          <svg class="svg-theme" width="18" height="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
-            <circle class="fill-theme" opacity=".16" cx="9" cy="9" r="9"></circle>
-            <circle class="fill-theme" cx="9" cy="9" r="3"></circle>
-            <circle class="fill-theme" opacity=".32" cx="9" cy="9" r="6"></circle>
-          </svg>
-        <!-- /ko -->
-      </div>
-      <div class="calls-controls">
-
-        <!-- ko if: is_group() && call().state() === z.calling.enum.CallState.ONGOING && call().self_user_joined() -->
-          <div class="calls-controls-row calls-controls-row-participants"
-               data-bind="click: $parent.click_on_conversation">
-            <!-- ko foreach: call().participants() -->
-              <user-avatar class="user-avatar-xs" params="user: $data.user"></user-avatar>
-            <!-- /ko -->
-          </div>
-        <!-- /ko -->
-
-        <div class="calls-controls-row calls-controls-row-conversation"
-             data-bind="click: $parent.click_on_conversation">
-          <div class="name"
-               data-bind="attr: {'data-uie-uid': id, 'data-uie-value': display_name()},
-                          css: {'text-theme': $parent.is_selected_conversation($data)},
-                          text: display_name()"
-               data-uie-name="item-call"></div>
+  <!-- ko if: show_calls()-->
```

Comment:
Will create a UI component for this next. Then we can always show the joined call while hiding what we do on the stream without duplicating hundreds of lines of code.